lib/minidoc/test_helpers.rb

```diff
@@ -8,15 +8,15 @@ def clear_database
     end
 
     def clear_collections
-      each_collection { |c| c.remove({}) }
+      each_collection(&:drop)
     end
 
     def clear_indexes
       each_collection(&:drop_indexes)
     end
 
     def each_collection(&block)
-      Minidoc.database.collections.
+      $mongo.collections.
```

Comment:
OK yep, it looks like this can simply be `Minidoc.connection` instead of `$mongo`